devel/dialog/Portfile

```diff
@@ -20,14 +21,36 @@ homepage            https://invisible-island.net/dialog/
 master_sites        ftp://ftp.invisible-island.net/pub/dialog/ \
                     https://invisible-mirror.net/archives/dialog/
 
-#distname   ${name}
-#dist_subdir    ${name}/${version}
-#worksrcdir ${name}-${version}
 extract.suffix      .tgz
 
-checksums           rmd160  b5728fdcca79db8c5e59bd9c4e1a16a6c40f6c50 \
-                    sha256  01c2d1e2e9af9b083ea200caad084fdfda55178d5bbf4e42c9fff44935151653 \
-                    size    561235
+checksums           ${distfiles} \
+                        rmd160  ae168047cc8c5a8c18cc81ac535bb3baff839e96 \
+                        sha256  1f62df6a48dac087b98452119e4cdfcaa3447b3eb5746b241e5632e1d57bfc4b \
+                        size    562950
+
+gpg_verify.use_gpg_verification \
```

Comment:
I personally don't really see the need of adding all of this extra lines of code; what's wrong with just the checksums like every other port does?

Reply:
Checksums are effective to avoid replacing an archive with the same name. From my observation, this happens when hotfixes are merged into a tag without a version increase.

GPG verification protects against `master_sites` compromise. The author publishes the signature along with the archive. In any case, it is better to make sure that the archive has not been tampered with during the new release in `master_sites`.

Reply:
I understand that part; I just don't think it's enough of a reason to complicate Portfiles with all this additional code

Reply:
Done, I agree.